src/hooks/useCrc8.ts

```diff
@@ -0,0 +1,5 @@
+import { useMemo } from "react";
+import { calculateCrc8 } from "../utils/calculateCrc8";
+
+export const useCrc8 = (data?: Uint8Array) =>
+  useMemo(() => calculateCrc8(data), [data]);
```

Comment:
すみません、書き漏らしがありました。ここで`useMemo`を使っているので、`calculateCrc8`を`undefined`も受け入れるようにする必要はありません。
```suggestion
export const useCrc8 = (data?: Uint8Array) =>
  useMemo(() => data && calculateCrc8(data), [data]);
```
のようにすれば十分です。